### ISSUE TITLE
minimap zoom is confusing

### ISSUE THREAD

**Issue by nilclass** [apollo-ng/dspace-app-apollong#38]
When I zoom in on the big map, the minimap zooms out. When I zoom out on the big map, the minimap zooms in. This seems to be intended behaviour, but I can't see how it's useful.

I would expect the minimap to give me some context to interpret the big map. So when I am at a zoomlevel where I see villages, I may peek at the minimap to figure out which towns are nearby and in which direction. This works so far - however when I zoom out a little further the zoomlevel barely changes, so the minimap becomes useless (since it shows _less_ area than the main map). Now when I zoom out even further (e.g. to go to another country), the minimap becomes really ridiculous. While I'm dragging the map around, the minimap shows me little villages in an area that I can't even identify on the big map (also, as a sidenote: the minimap is centered to the center of the browser window, not the center of the actual map).

Please explain to me why it is the way it is -- or change it, so it makes sense :)

**Comment by yggi** [comment]
I agree, at the moment the minimap is only useful when the user is zoomed in on the big map.

@chron0 suggested that the minimap shows the surrounding of the feature, zoomed in, when a feature-modal is open. (which hides the big map). I like that.

Also we discussed a kind of inverted behavior of the minimap, so that it zooms in the more you zoom out on the big map, but personally I think that would be not really useful and confusing.

Maybe the minimap should just zoom out even more? Like constantly n zoomlevels lower than the big map?

**Comment by yggi** [comment]
@nilclass how do you like the new behavior?

**Comment by elf-pavlik** [comment]
@yggi, @chron0 claims "done! except touch events _drag_ on mobile" .... @nilclass wanna take a look at current behavior since you submitted this issue?

**Comment by nilclass** [comment]
I like the new behavior and would consider this issue done. The only thing I still noticed is that the bounding box indicator (which totally rocks to have btw) shows the bounds of the entire map, not just the visible parts. I don't know how hacky it would be to determine what parts of the map are actually visible (i.e. not overlapped by any controls) and only show those bounds on the minimap. Or, even more awesome: grey out the areas of the bounding box indicator that are overlapped by UI elements. But those are just Spielereien :)

-> close ?

**Comment by chron0** [comment]
I think the confusion here is somewhere else. The bounding box as well as the minimap always show the correct map view. However, when you have the sidebar open, a part of the map is not visible (although it is there in the background and of course in fullscreen, since the basemap always has to be the full window size). Every other element is OVER the basemap, since dynamically changing the size of the basemap doesn't work. That might lead to this confusion :)

**Comment by nilclass** [comment]
Why doesn't changing the size of the basemap work? It happens when you change the size of the window.
